services/src/test/scala/cromwell/pipeline/service/GitLabProjectVersioningTest.scala

```diff
@@ -38,13 +38,41 @@ class GitLabProjectVersioningTest extends AsyncWordSpec with ScalaFutures with M
           _ shouldBe Right(project)
         }
       }
-      "throw new VersioningException with not 400 response" taggedAs Service in {
+      "throw new VersioningException with 400 response" taggedAs Service in {
         when(request(activeProject)).thenReturn(Future.successful(Response(StatusCodes.BadRequest.intValue, "", Map())))
         gitLabProjectVersioning.createRepository(activeProject).map {
           _ shouldBe Left(VersioningException("The repository was not created. Response status: 400"))
         }
       }
     }
+
+    "getProjectVersions" should {
+      lazy val dummyVersionsJson: String = s"[${Json.stringify(Json.toJson(dummyVersion))}]"
```

Comment:
You may leave it as is, but I think the following code is a bit better
`Json.stringify(Json.toJson(Seq(dummyVersion))`
Also, no need for a `lazy` modifier.

Reply:
Lazy removed.